spirv_msl.hpp

```diff
@@ -37,6 +37,8 @@ struct MSLVertexAttr
 	uint32_t msl_offset = 0;
 	uint32_t msl_stride = 0;
 	bool per_instance = false;
+	bool uint8 = false;
```

Comment:
Why two fields instead of having an enum?

Reply:
Because it was originally one boolean indicating an unsigned format. But yeah, these two are mutually exclusive, and should probably use an enum. I'll send a fix.